MycroftAI/mycroft-core
# Upgrade requests to 2.25.1

## Description
Upgrades requests to 2.25.1

## How to test
Run the unit tests.
Verify with a clean installation that all things that get downloaded still work, and network connectivity is still correct.

## Contributor license agreement signed?
CLA [x]
